### ISSUE TITLE
Multiple database driver adaptations to do list

### ISSUE THREAD

**Issue by manyuanrong** [manyuanrong/dso#4]
In order to achieve multiple database driven adaptation, we need to implement the following

- [x] **src/drivers/base.ts** Unified interface definition
- [x] **src/drivers/mysql.ts** MySQL adapter based on base.ts
- [ ] **src/drivers/postgres.ts** Postgres adapter based on base.ts
- [ ] **src/dso.ts** Do not import any driver, dynamically import the driver through the parameter of connect
- [ ] **mysql.ts** At the same level as mod.ts, the export directly imports dso of MySQL driver
- [ ] **postgres.ts** At the same level as mod.ts, the export directly imports dso of Postgress driver

**Comment by evnix** [comment]
This would be amazing!

**Comment by tksilicon** [comment]
Do you have anything you have done, so I can take it from there, you can share the code in a feature branch if you wouldn't commit on master so I can finish it? @manyuanrong?

**Comment by manyuanrong** [comment]
@tksilicon The items in the todo list are still not in progress. If you want, I can add you as a collaborator.

**Comment by tksilicon** [comment]
You can @manyuanrong. This issue can be resolved in a week or two. I very much am interested in this project. It has been a learning opportunity for me.

**Comment by manyuanrong** [comment]
@tksilicon I have sent an invitation. I hope to do this work on a new branch and finally merge into master

**Comment by tksilicon** [comment]
Definitely. It will be a new branch by me to merge into master by you. @manyuanrong
